api/report_server.thrift

```diff
@@ -170,18 +177,18 @@ service codeCheckerDBAccess {
                        1: i64 reportId)
                        throws (1: shared.RequestFailed requestError),
 
-  // get the results for one runId
+  // get the results for some runIds
   ReportDataList getRunResults(
-                               1: i64 runId,
+                               1: list<i64> runIds,
```

Comment:
Now the `ReportFilter` struct looks like this.
```thrift
struct ReportFilter{
  1: optional string          filepath,
  2: optional string          checkerMsg,
  3: optional shared.Severity severity,
  4: optional string          checkerId,
  5: optional string          bugHash
}
```
If the user sets for example multiple severity levels (high, critical), we have to create and send multiple ReportFilter items at the thrift (list of ReportFilters) where the value of every other field is 'null'.

I am thinking on that to replace this struct with the following one where `every field item is a list`. It would be better to `send the run ids and review statuses in this struct`, because it's part of the filter. 
In this case we would send only one object of this struct which would contain every necessary information for filtering.
```thrift
struct ReportFilter{
  1: optional list<i64>                 runIds,
  2: optional list<string>              filepath,
  3: optional list<string>              checkerMsg,
  4: optional list<shared.Severity>     severity,
  5: optional list<shared.ReviewStatus> reviewStatus,
  5: optional list<string>              checkerId,
  6: optional list<string>              bugHash
}
```